test/app.spec.js

```diff
@@ -3,6 +3,8 @@ const fs = require('fs'),
   models = require('../app/models'),
   path = require('path');
 
+jest.mock('../app/services/emails');
```

Comment:
Mm i don't like have all mocks in app.spec.js, can we try something like this in signup test ? 
`emailService.sendMail = jest.fn().mockResolvedValueOnce('Nice');`